src/main/resources/view/LightTheme.css

```diff
@@ -63,12 +63,12 @@
     -fx-text-fill: white;
 }
 
-#remark {
+.remark {
     -fx-hgap: 7;
     -fx-vgap: 3;
 }
 
-#remark .label {
+.remark {
     -fx-background-color: #6FB6ED; /*light blue for tags*/
```

Comment:
Hmm would it be possible to combine these two?

Reply:
Yea definitely!